accepted/0000-workspaces-command.md

```diff
@@ -0,0 +1,31 @@
+- Start Date: 2017-05-04
+- RFC PR:
+
+# Summary
+
+This document specify a new command, that can be used to execute subcommands inside a project workspaces.
+
+# Motivation
+
+With the addition of the Workspace feature, it will become handy to be able to execute commands inside other workspaces than the one in the current directory.
+
+# Detailed design
+
+This RFC suggests to add the two following commands:
+
+```
+$> yarn workspace <project-name> pwd
+$> yarn workspace <project-name> <command-name> ...
```

Comment:
The other thing that we can do to make this useful cross-platform is to specify the `$PATH` the way `package.json#scripts` do.

From #62:

> There's three options for what the `$PATH` should be within workspaces.
> 
> - The project's `node_modules/.bin`
> - The workspace's `node_modules/.bin`
> - Both (w/ workspace before project)